solution02/src/test/resources/user.properties

```diff
@@ -17,16 +17,16 @@
 #
 
 #server
-severHost=localhost
+severHost=google.idp.com
 severPort=9443
 
 # Tomcat
-tomcatHost=localhost
+tomcatHost=10.100.7.29
 tomcatPort=8080
 
 # User Management
-admin_username=admin
-admin_password=admin
+adminUsername=admin
+adminPassword=admin
```

Comment:
Password should not be uploaded. It would be better to keep this blank and have a doc (such as in the README.md) to explaining this

Reply:
@Niranjan-K  This due to the test grid. 
Mail thread initiated on your concern